<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1030</width>
    <height>676</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>13</y>
      <width>651</width>
      <height>461</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>649</width>
       <height>459</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="leftMargin">
       <number>13</number>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1030</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actAddFold"/>
   <addaction name="actAddFile"/>
   <addaction name="actDelItem"/>
   <addaction name="actTraverse"/>
   <addaction name="separator"/>
   <addaction name="actBig"/>
   <addaction name="actSmall"/>
   <addaction name="actRealSize"/>
   <addaction name="separator"/>
   <addaction name="actionWidth"/>
   <addaction name="actHeight"/>
   <addaction name="separator"/>
   <addaction name="actFloat"/>
   <addaction name="actView"/>
   <addaction name="actExit"/>
  </widget>
  <widget class="QDockWidget" name="dockWidget">
   <property name="windowTitle">
    <string>图片目录树</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="spacing">
      <number>4</number>
     </property>
     <property name="leftMargin">
      <number>1</number>
     </property>
     <property name="topMargin">
      <number>2</number>
     </property>
     <property name="rightMargin">
      <number>4</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QTreeWidget" name="treeWidget">
       <property name="columnCount">
        <number>2</number>
       </property>
       <attribute name="headerMinimumSectionSize">
        <number>30</number>
       </attribute>
       <attribute name="headerDefaultSectionSize">
        <number>150</number>
       </attribute>
       <column>
        <property name="text">
         <string>节点</string>
        </property>
        <property name="font">
         <font>
          <pointsize>9</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="background">
         <color>
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </property>
        <property name="foreground">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </property>
       </column>
       <column>
        <property name="text">
         <string>文件类型</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actAddFold">
   <property name="icon">
    <iconset resource="rc.qrc">
     <normaloff>:/icons/delfold1.bmp</normaloff>:/icons/delfold1.bmp</iconset>
   </property>
   <property name="text">
    <string>添加目录</string>
   </property>
   <property name="toolTip">
    <string>添加目录</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+F</string>
   </property>
  </action>
  <action name="actAddFile">
   <property name="icon">
    <iconset resource="rc.qrc">
     <normaloff>:/icons/305.bmp</normaloff>:/icons/305.bmp</iconset>
   </property>
   <property name="text">
    <string>添加文件</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actDelItem">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="rc.qrc">
     <normaloff>:/icons/324.bmp</normaloff>:/icons/324.bmp</iconset>
   </property>
   <property name="text">
    <string>删除节点</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actTraverse">
   <property name="icon">
    <iconset resource="rc.qrc">
     <normaloff>:/icons/314.bmp</normaloff>:/icons/314.bmp</iconset>
   </property>
   <property name="text">
    <string>遍历节点</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="actBig">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="rc.qrc">
     <normaloff>:/icons/418.bmp</normaloff>:/icons/418.bmp</iconset>
   </property>
   <property name="text">
    <string>放大</string>
   </property>
   <property name="shortcut">
    <string>Up</string>
   </property>
  </action>
  <action name="actSmall">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="rc.qrc">
     <normaloff>:/icons/416.bmp</normaloff>:/icons/416.bmp</iconset>
   </property>
   <property name="text">
    <string>缩小</string>
   </property>
   <property name="shortcut">
    <string>Down</string>
   </property>
  </action>
  <action name="actRealSize">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="rc.qrc">
     <normaloff>:/icons/preview1.bmp</normaloff>:/icons/preview1.bmp</iconset>
   </property>
   <property name="text">
    <string>实际大小</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionWidth">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="rc.qrc">
     <normaloff>:/icons/165.JPG</normaloff>:/icons/165.JPG</iconset>
   </property>
   <property name="text">
    <string>适合宽度</string>
   </property>
  </action>
  <action name="actHeight">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="rc.qrc">
     <normaloff>:/icons/103.bmp</normaloff>:/icons/103.bmp</iconset>
   </property>
   <property name="text">
    <string>适合高度</string>
   </property>
  </action>
  <action name="actFloat">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="rc.qrc">
     <normaloff>:/icons/814.bmp</normaloff>:/icons/814.bmp</iconset>
   </property>
   <property name="text">
    <string>窗体浮动</string>
   </property>
  </action>
  <action name="actView">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset theme="audio-volume-high" resource="rc.qrc">
     <normaloff>:/icons/preview1.bmp</normaloff>:/icons/preview1.bmp</iconset>
   </property>
   <property name="text">
    <string>窗体可见</string>
   </property>
  </action>
  <action name="actExit">
   <property name="icon">
    <iconset resource="rc.qrc">
     <normaloff>:/icons/exit.bmp</normaloff>:/icons/exit.bmp</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="rc.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>483</x>
     <y>297</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
